docs/tutorials/ua_fix_scenarios.md

```diff
@@ -0,0 +1,365 @@
+# Using fix command to solve CVE/USNs on the machine
+
+The Ubuntu Advantage Tools (UA) program can be used to inspect and resolve
+[CVEs](https://ubuntu.com/security/cves) and [USNs](https://ubuntu.com/security/notices) (Ubuntu
+Security Notices) on this machine.
+
+Every CVE/USN is fixed by trying to upgrade all of the affected packages described by the CVE or
+USN. Sometimes, the packages fixes can only be applied if an UA service is already enabled on the
+machine.
+
+On this tutorial, we will cover the main scenarios that can happen when running the UA fix command.
+
+## Prerequisites
+
+On this tutorial, we will use [LXD](https://linuxcontainers.org/lxd/) containers.
+To set up LXD on your computer, please follow this
+[guide](https://linuxcontainers.org/lxd/getting-started-cli/).
+
+## Creating the Xenial LXD container
+
+To test the `ua fix` command, let's create a Xenial LXD container. Remember to set up LXD as
+mentioned on the [Prerequisites](#prerequisites) section. After that, just run the command:
+
+```console
+$ lxc launch ubuntu-daily:xenial dev-x
+```
+
+After running that, let's access the container by running:
+
+```console
+$ lxc shell dev-x
+```
+
+Every time we say: "run the command" our intention will be for
+you to run that command on your Xenial LXD container.
+
+## Using UA fix
+
+First, let's see what happens on your system when `ua fix` runs. We will choose
+to fix a CVE that does not affect the Xenial container,
+[CVE-2020-15180](https://ubuntu.com/security/CVE-2020-15180). This CVE address security
+issues for the `MariaDB` package, which is not installed on the system. Let's confirm that
+it doesn't affect the system by running this command:
+
+```console
+$ ua fix CVE-2020-15180
+```
+
+You should see an output like this one:
+
+```
+CVE-2020-15180: MariaDB vulnerabilities
+https://ubuntu.com/security/CVE-2020-15180
+No affected source packages are installed.
+✔ CVE-2020-15180 does not affect your system.
+```
+
+Every `ua fix` output will have a similar output structure where we describe the CVE/USN,
+display the affected packages, fix the affected packages and at the end, show if the
+CVE/USN is fully fixed in the machine.
+
+You can better see this on an `ua fix` call that does fix a package. Let's install a package
+on the container that we know are associated with [CVE-2020-25686](https://ubuntu.com/security/CVE-2020-25686).
+You can install that package by running this command:
+
+```console
+$ sudo apt install dnsmasq=2.75-1
```

Comment:
I think you might need an `apt update` before this will work in a fresh container

Reply:
Good catch